superset-frontend/cypress-base/cypress/integration/dashboard/controls.test.js

```diff
@@ -0,0 +1,122 @@
+/**
```

Comment:
Are these changes only file renames? It's tough to tell from the diff

Reply:
For anything other than `visualizations` tests, it's just file renames. Had to fix a couple of visualization tests, will add notes shortly.